### DISCUSSION TITLE
Custom Background Images

### DISCUSSION THREAD

**Discussion by byeoon** [community/community#127763] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Will there ever be a feature for custom wallpapers on your profile? Instead of just having a basic black / basic white background, having the ability to upload an image would make profiles feel a lot more nice.

I personally think it would look fantastic if profiles look like this. These were aesthetically pleasing and could give a little more information about someone by how they theme their profiles.
![image](https://github.com/community/community/assets/47872200/61c9050f-9455-4637-990c-93ad0c366305)
![image](https://github.com/community/community/assets/47872200/ac00e5b2-67dc-4a02-8845-79fbe5097140)

```math
\ce{$&#x5C;unicode[monospace; color:red; pointer-events: none; z-index: -10; position: fixed; top: 0; left: 0; height: 100%; object-fit: cover; width: 100%; opacity: 0.4; background-size: cover; background-image: url('https://github.com/byeoon/byeoon/assets/47872200/b763f486-7834-49b3-985b-0793ea0ca58a');]{x0000}$}
```

**Comment by Vendicated** [comment]
would go crazy fr

```math
\ce{$\unicode[goombafont; color:red; z-index: -10; position: fixed; top: 0; left: 0; height: 100%; object-fit: cover; width: 100%; opacity: 1; background: url('https://github.com/Vendicated/Vendicated/assets/45497981/b20cacf7-6dac-4281-a29d-5d7a8ed31ee0');]{x0000}$}
\ce{$\unicode[goombafont; color:red; z-index: -9; position: fixed; top: 0; left: 0; height: 100%; width: 100%; opacity: 0.8; background: var(--bgColor-default);]{x0000}$}
```

**Reply by Vendicated** [reply]
NOOOO THEY PATCHED IT :(

**Reply by ggtylerr** [reply]
bad vendi, no xss for u >:(

**Comment by mugman174** [comment]
would be nice if it was a feature instead of a bug

(it's no longer a bug)

**Comment by Bims-sh** [comment]
I would approve of this 
```math
\ce{$\unicode[goombafont;color:red;pointer-events: none;z-index: -1;position: fixed;top: 100px;left: 0;height: 100vh;object-fit: cover;width: 100vw;opacity: 0.5;background: url('https://github.com/Bims-sh/Bims-sh/assets/46683337/33e9bca2-a102-4168-8817-61add445b15f');align-content: center;background-repeat: repeat;background-size: 20%;]{x0000}$}

**Comment by byeoon** [comment]
shikono noko no ko koshitantan cubes AND SHIGGY omg omg omg awesome




```math
$$\ce{$\unicode[comic-sans; z-index: 99999; position: fixed; top: 0; left: 0; height: 100%; width: 100% ;pointer-events: none; opacity: 1; background: url('https://github.com/byeoon/byeoon/assets/47872200/c69f3545-890b-4b92-9f6d-94039034b414');background-size:100% 100%;]{x0000}$}$$
```

**Comment by ghost** [comment]
Downside is, this can be laggy for some systems. 
I've recently posted about this on Mastodon: https://m.sudovanilla.org/#/social.vivaldi.net/s/112578537311557003

**Reply by Bims-sh** [reply]
Well if they officially implement this it will be stripped down ig

**Reply by mugman174** [reply]
maybe this could be a setting that users on those systems could disable

**Reply by Bims-sh** [reply]
opt in could be cool

**Reply by byeoon** [reply]
Could probably disable it under accessibility or something if it was a feature

**Comment by byeoon** [comment]
They patched it NOOOOOOOOOOOOO

**Comment by Pandaptable** [comment]
soon:tm: (copium)

**Comment by Bims-sh** [comment]
To be expected sadly, hope this thread gets noticed so they may actually consider adding this as a feature :)

**Reply by byeoon** [reply]
I hope so too

**Comment by Vap0r1ze** [comment]
sadness

**Comment by byeoon** [comment]
Oh also having custom effects like blur, shadows, etc would be fantastic
